src/schema/__tests__/film.js

```diff
@@ -99,7 +99,7 @@ describe('Film type', async () => {
       '{ allFilms { edges { cursor, node { ...AllFilmProperties } } } }',
     );
     const result = await swapi(query);
-    expect(result.data.allFilms.edges.length).to.equal(7);
+    expect(result.data.allFilms.edges.length).to.equal(6);
```

Comment:
Can you investigate why we have one less film?
It should be the opposite.

Reply:
Can you please check if it related to https://github.com/graphql/swapi-graphql/pull/172 or happened for some other reason?

Reply:
I think that there were only 6 movies in original too. I checked the [data source file](https://github.com/phalt/swapi/blob/master/resources/fixtures/films.json
) from `swapi.co`

This makes sense if we look at commit history of the file,  `Episode VII – The Force Awakens` was released in 2015 and the last time the owner update things was 6 years ago (2014) and didn’t really touched them (except to fix typos) so it makes sense since there were only 6 movies.

`swapi.dev` has issue open https://github.com/Juriy/swapi/issues/8 to update movies